hedyorg/hedy
# [FEAT] responsive profile page

**Description**

Makes the profile page look better on smaller-width screens.

Also replaces `generated.css` with the correct file, the output of the `generate-development-css` script.

**Fixes #3158 **

**How to test**

1. Go to http://localhost:8080/my-profile
1. Make your profile page public
1. Go to your profile page
1. Change the width of your browser window
1. Verify that nothing looks out of place

**Checklist**


Screenshots:

~800px width
![Screenshot from 2023-03-15 21-40-26](https://user-images.githubusercontent.com/602877/225520108-0143ca56-9587-421b-a65b-468b63d52cde.png)

~1900px width
![Screenshot from 2023-03-15 21-39-41](https://user-images.githubusercontent.com/602877/225520138-7824d4fb-98b5-458e-82ce-289b1bf7a192.png)

~500px width 1/2
![Screenshot from 2023-03-15 21-37-25](https://user-images.githubusercontent.com/602877/225520326-1552f2f2-c6ed-4703-832a-1a3efb830cce.png)


~500px width 2/2
![Screenshot from 2023-03-15 21-37-42](https://user-images.githubusercontent.com/602877/225520350-1705da1b-27f8-4251-91f4-3c74ed6e4927.png)
